services/base-jdbc/src/main/java/org/eclipse/hono/service/base/jdbc/config/JdbcOptions.java

```diff
@@ -60,6 +60,27 @@ public interface JdbcOptions {
      */
     OptionalInt maximumPoolSize();
 
+    /**
+     * Gets the minimum size of the DB connection pool.
+     *
+     * @return The minimum number of connections in the pool.
+     */
+    OptionalInt minimumPoolSize();
+
+    /**
+     * Gets the initial size of the DB connection pool.
+     *
+     * @return The initial number of connections in the pool.
+     */
+    OptionalInt initialPoolSize();
+
+    /**
+     * Gets the maximum idle time of connections in the DB connection pool.
+     *
+     * @return The maximum idle time of connections in the pool.
+     */
+    OptionalInt maximumIdleTime();
```

Comment:
should we add checks to ensure the values are >= 0?

Reply:
sure, we could do that. Is there an annotation or something you have in mind that I should use or why did you comment this in this interface rather than in the JdbcProperties class?